pkgs/by-name/js/json2cdn/package.nix

```
@@ -0,0 +1,57 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  makeWrapper,
+  gradle_8,
+  jre_headless,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "json2cdn";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "uku3lig";
+    repo = "json2cdn";
+    tag = finalAttrs.version;
+    hash = "sha256-pHmzeZHZyr3FyfkXwrdPk+lcHQKH6t4pnDD9ImMgSV8=";
+  };
+
+  nativeBuildInputs = [
+    gradle_8
+    makeWrapper
```

```suggestion
    makeBinaryWrapper
```

Nitpick. There's a small bit of overhead with the script based (default) `makeWrapper`